lib/workers/repository/update/branch/index.ts

```diff
@@ -355,10 +355,19 @@ export async function processBranch(
       dependencyDashboardCheck === 'rebase' ||
       !!config.dependencyDashboardRebaseAllOpen ||
       !!config.rebaseRequested;
-
+    const userApproveAllPendingPR = !!config.dependencyDashboardAllPending;
+    const userOpenAllRateLimtedPR = !!config.dependencyDashboardAllRateLimited;
     if (userRebaseRequested) {
       logger.debug('Manual rebase requested via Dependency Dashboard');
       config.reuseExistingBranch = false;
+    } else if (userApproveAllPendingPR) {
+      logger.debug(
+        'Manual Approve All Pending PRs requested via Dependency Dashboard'
```

Comment:
This is hard to parse for me, can we try something like:

```suggestion
        'A user manually approved all pending PRs via the Dependency Dashboard.'
```